muspinsim_input/muspinsim_input.xml

```diff
@@ -0,0 +1,423 @@
+                       optional="true" value="inf"
+                       help="Optional, (default is 'inf' - infinity).
+                       Allows default expressions, default constants, and special 'range()' function (see help section).
+                       Warning: both density matrices and dissipative couplings for finite temperatures are only calculated approximatively, see muspinsim docs."/>
+            </repeat>
+        </section>
+        <section name="fitting_params" expanded="true" title="Fitting Parameters">
+            <conditional name="fitting_options">
+                <param name="fitting" type="select" display="radio" optional="false" value="" label="Fit experimental data with simulations"
+                       help="Fitting requires a file with data to fit. File must be given in muspinsim tool, or by manually setting filepath for keyword 'fitting_data' if running muspinsim externally">
+                    <option value="true">Yes</option>
+                    <option selected="true" value="">No</option>
+                </param>
+                <when value="true">
+                    <param name="fitting_method" type="select" display="radio" optional="false" value="nelder-mead" label="Method to use to fit the data"
+                           help="Currently available are only nelder-mead (default) and lbfgs">
+                        <option selected="true" value="nelder-mead">nelder-mead</option>
+                        <option value="lbfgs">lbfgs</option>
+                    </param>
+                    <repeat name="fitting_variables" title="Variable to fit to the experimental data">
+                        <param name="var_name" type="text" optional="false" label="Name of the variable"/>
+                        <param name="start_val" type="text" value="0" label="Starting value"
+                               help="Optional, default is 0. Each entry allows default expressions, constants (including MHz, muon_gyr) and functions but cannot contain names of other variables"/>
+                        <param name="min_bound" type="text" value="-inf" label="minimum bound"
+                               help="Optional, default is -inf"/>
+                        <param name="max_bound" type="text" value="inf" label="maximum bound"
+                               help="Optional, default is inf"/>
+                    </repeat>
+                    <param name="fitting_tolerance" type="float" optional='true' value="" label="Tolerance for the fitting."
+                           help="Used as the tol parameter in Scipy's scipy.optimize.minimize method. Does not accept expressions/functions/constants"
+                    />
+                </when>
+                <when value=""/>
+            </conditional>
+        </section>
+    </inputs>
+    <outputs>
+        <data format="txt" name="out_file" from_work_dir="outfile.in" />
```

Comment:
This would be better with a custom label based on the `out_file_prefix` parameter - otherwise the user never actually sees the value of that parameter in the Galaxy UI. Plus if you set up multiple configs you want to be able to distinguish them!